### ISSUE TITLE
Importing module gives warning about unapproved verbs

### ISSUE THREAD

**Issue by lahell** [lahell/PSDiscoveryProtocol#2]
Importing this module gives the following warning:

```
WARNING: The names of some imported commands from the module 'PSDiscoveryProtocol' include unapproved verbs that might make them less discoverable. To find the commands with
 unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
```

Will consider using Invoke and Get instead of Capture and Parse.

**Comment by lahell** [comment]
This issue is fixed in the `feature-export-pcap` branch by commit 621c205.

**Comment by lahell** [comment]
Fixed by commit 9db80bd4b44e4d0ac10e0cda77d18fb222c992b2.
